src/assets/i18n/de.json

```diff
@@ -266,11 +266,11 @@
       "ALL": "Alle",
       "CATEGORY_FILTER": "Filtern nach",
       "CONSENT_HEADER": "Art der Datennutzung",
-      "CONSENT_CENTRAL_TOOLTIP": "For the central analysis you will recieve patient data pseudonymised by the MII, which fit your research project proposal. Pseudonymised data can only be used, if patients have consented to their use.The researches will recieve the pseudonymised data in form of a Download. They will then be able to conduct their analysis with greater flexibility and depths as well as faster speed compared to decentral analyis. The collection of the consent is currently underway across all sites. However it can be expected, that for most queries less patients will be found for central than de-central analysis.",
-      "CONSENT_DISTRIBUTED_TOOLTIP": "  For distributed analyses the analysis is carried out at the sites. The researchers recieve anonymous result data. For many sites a de-central analysis can be based on a different legal other than a consent. This often leads to larger numbers of patients found, which fulfill the requirements for your research proposal. Some sites also require the consent of a patient for feasibility queries. These sites will answer feasibility queries with comparatively lower number. The actual number of patients available from these site for distributed analyses might be higher. The actual number of available patients will not be clear until after the research proposal has been processed..",
-      "CONSENT_CENTRAL": " is assumed (Data consolidated centrally)",
+      "CONSENT_CENTRAL_TOOLTIP": "For the central analysis you will recieve patient data pseudonymised by the MII, which fit your research project proposal. Pseudonymised data can only be used, if patients have consented to their use.The researchers will recieve the pseudonymised data in form of a download. They will then be able to conduct their analysis with greater flexibility and depth as well as faster speed compared to decentral analyis. The collection of consents is currently ongoing across all sites. However it can be expected that for most queries less patients will be found for central than de-central analysis.",
```

Comment:
"(...)if patients have consented to their use.The researchers(...)" - there should be a blank after the full stop.